Improve documentation (#1765)
onflow/cadence: docs/language/core-events.md

Event name: `flow.AccountContractUpdated`

```cadence
pub event AccountContractUpdated(
    address: Address,
    codeHash: [UInt8],
    contract: String
)
```

| Field       | Type      | Description                                              |
| ----------- | --------- | -------------------------------------------------------- |

> | `address`   | `Address` | The address of the account the contract gets updated on  |

Reviewer: ```suggestion
| `address`   | `Address` | The address of the account where the updated contract is deployed  |
```
The wording is a bit awkward here.
